config/initializers/devise.rb

```diff
@@ -4,10 +4,15 @@
 GITHUB_APP_ID     = Rails.env.production? ? ENV['GITHUB_APP_ID']     : 'cc17063f6602cc4878ff'
 GITHUB_APP_SECRET = Rails.env.production? ? ENV['GITHUB_APP_SECRET'] : 'e93988593b6f7bfa1d0ff2cf10438ee3e547f8a4'
 
+# Set this to "https://rgsoc-teams-pr-oauth-proxy.herokuapp.com/oauth-proxy" for staging and review apps,
+# leave blank to stick with OmniAuth's default of "<scheme>://<host>/users/auth/github/callback":
+GITHUB_OAUTH_CALLBACK_URI = ENV['GITHUB_OAUTH_CALLBACK_URI']
```

Comment:
Not necessarily related to this PR, but do you think it would be worth refactoring this to a classic `dotenv` setup?

We could include an example `.env.example` file and developers could adjust it to their needs?

Doing so, we could remove all the `Rails.env.production` things and simply use:
```ruby
GITHUB_APP_ID = ENV.fetch('GITHUB_APP_ID')

# or if it's really really optional:
GITHUB_APP_ID = ENV.fetch('GITHUB_APP_ID', nil)
```

Using `fetch` would have the benefit that the app would already crash when attempting to start instead of failing at some random point in the future.

For CI, there's also the possibility to encrypt environment variables - works like charm ✅ 

What do you think?